x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/apm_403_response_to_a_post.json

```diff
@@ -7,7 +7,8 @@
     "Security scans and tests may result in these errors. Misconfigured or buggy applications may produce large numbers of these errors. If the source is unexpected, the user unauthorized, or the request unusual, these may indicate suspicious or malicious activity."
   ],
   "index": [
-    "apm-*-transaction*"
+    "apm-*-transaction*",
+    "traces-apm*"
```

Comment:
Please make these changes in https://github.com/elastic/detection-rules

Reply:
For this file:
https://github.com/elastic/detection-rules/blob/main/rules/apm/apm_403_response_to_a_post.toml

Reply:
PR up: https://github.com/elastic/detection-rules/pull/1335

Reply:
Thanks! You can back out these prepackaged_rules changes now.
We'll bring them back shortly in another PR, and handle things like bumping the version, etc.

Reply:
Done: https://github.com/elastic/kibana/pull/105334/commits/aaa5b4024e57e476746b977804373c98c1642d5b